src/utils.js

```diff
@@ -655,3 +655,69 @@ function binomialCoefficient(a, b) {
 	var denominator = facts[a - b] * facts[b];
 	return numerator / denominator;
 } 
+
+function getCallNameByType(type) {
+	switch (type) {
+		case 1: return "discard";
+		case 2: return "chi";
+		case 3: return "pon";
+		case 4: return "kan(ankan)";
+		case 5: return "kan(daiminkan)";
+		case 6: return "kan(shouminkan)";
+		case 7: return "riichi";
+		case 8: return "tsumo";
+		case 9: return "ron";
+		case 10: return "kyuushu kyuuhai";
+		case 11: return "kita";
+		default: return type;
+	}
+}
+
+//Get Emoji str by tile name
+function getTileEmoji(name) {
+	switch (name) {
```

Comment:
I think putting the Emojis into a 2-dimensional array (10x4) would make this function way shorter. Something like this:
```
var emojiList = [
["red🀋", "🀇", "🀈", ........],
[........],
[........],
[........]];
return emojiList[tile.type][tile.index];
```

Reply:
That sounds good, I will change it!